index.html

```diff
@@ -200,13 +224,21 @@ <h2 class="playfairFont">Get Started</h2>
     <p class="loremStyle">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
```

Comment:
- [ ] Please make sure the elements in the contact-me section are aligned properly as specified in the [Figma design 🎨](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=STzDZgG5tHGW7XUO-0). 
- [ ] **Hint:** You can add a separate container for the heading, the text, and the form and give it a limited width and align it to the center. The heading should also be aligned to the center.
    - Figma design:
      ![gRLsROsxoT](https://user-images.githubusercontent.com/104968236/229291217-6742bc46-f8ff-4a61-960b-6ebe18230e9f.png)
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/229291279-8b7e8da2-ff06-4107-8f3e-9cd5e85de8f0.png)